src/Assets/Config.php

```diff
@@ -96,11 +96,28 @@ public static function is_group_path_using_asset_directory_prefix( string $group
 	 * @return void
 	 */
 	public static function add_group_path( string $group_path_slug, string $root, string $relative, bool $is_using_asset_directory_prefix = false ): void {
-		static::$group_paths[ $group_path_slug ] = [
-			'root'     => self::normalize_path( $root ),
-			'relative' => trailingslashit( $relative ),
-			'prefix'   => $is_using_asset_directory_prefix,
-		];
+		/**
+		 * Allows for the group path to be filtered.
+		 *
+		 * This filters allows plugins/themes using the library to modify the root path
+		 * to follow symlinks. The library itself can't handle this on its own,
+		 * because it's not possible to be aware of where the library is placed relative
+		 * to the actual WordPress plugins or themes directories.
+		 *
+		 * @since 1.4.9
+		 *
+		 * @param array  $group_path      The group path data.
+		 * @param string $group_path_slug The slug of the group path.
+		 */
+		static::$group_paths[ $group_path_slug ] = apply_filters(
+			'stellarwp/assets/group_path',
```

Comment:
@dpanta94 This filter is missing the hook prefix that is used by other filters in this library.